generate-stackbrew-library.sh

```diff
@@ -5,7 +5,7 @@ shopt -s globstar
 declare -A aliases
 aliases=(
 	[9.2-jre7]='jre7'
-	[9.3-jre8]='latest jre8'
+	[9.4-jre8]='latest jre8'
```

Comment:
Oh I should also say that we have a 9.3.x release coming out in the next week or two with some important bug fixes, so at the very least there should be at least one more 9.3.x docker release.